node/tests/hubController-testConfig.json

```diff
@@ -17,11 +17,11 @@
         "resource": "power",
         "resourceBlob": {"value":true}
     },
+    "subscriptionInfo": {
+        "callbackUrl": "http://drsubscriptiontest.azurewebsites.net/subs",
+        "key": "imalwaysangry"
+    },
     "subscription": {
```

Comment:
Can you confirm the current status on there wtih your changes, and then file separate issues tracking bugs that will still need to be fixed  here and reference them here in the file as a TODO? 

---
In reply to: [100192052](https://github.com/openT2T/cloud/pull/14#discussion_r100192052) [](ancestors = 100192052)